statick_tool/statick.py

```diff
@@ -332,6 +352,14 @@ def run(
         discovery_plugins = self.config.get_enabled_discovery_plugins(level)
         if not discovery_plugins:
             discovery_plugins = list(self.discovery_plugins)
+        # Get timing information for finding files for discovery plugins.
+        dummy_plugin = DiscoveryPlugin()
+        plugin_start = time.time()
+        dummy_plugin.find_files(package)
+        duration = format(time.time() - plugin_start, ".4f")
+        timing = Timing(package.name, "Find Files", "Discovery", duration)
```

Comment:
Nitpick: everything else in the timing table is lower case, this is the only thing with capitals in it.
```suggestion
        timing = Timing(package.name, "find files", "Discovery", duration)
```

Reply:
This would also be fine with me
```suggestion
        timing = Timing(package.name, "find_files", "Discovery", duration)
```